diagnostic.md

```diff
@@ -149,7 +177,8 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
+divOne thru divThree will be justified to the right of the screen,
+divFour will be justified on the left side.
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).